upup/pkg/fi/cloudup/scalewaytasks/instance.go

```diff
@@ -75,6 +75,37 @@ func (s *Instance) Run(c *fi.Context) error {
 	return fi.DefaultDeltaRunMethod(s, c)
```

Comment:
Why is the file called "instance"? I thought the Scaleway terminology is "server"?

Reply:
Technically you're right, "instance" is the name of our virtual machine product so it is related to servers, volumes, images and snapshots but we use interchangeably the terms server and instance. Would you prefer the term server ?

Reply:
I prefer which one you think it's more appropriate.

Reply:
...but please consistently use the same term for the same concept.